components/divider/stories/divider.stories.js

```diff
@@ -44,18 +46,83 @@ export default {
 		rootClass: "spectrum-Divider",
 		size: "m",
 		vertical: false,
-		customStyles: {
-			"min-inline-size": "200px",
-		},
 	},
 	parameters: {
 		componentVersion: version,
 	},
 };
 
+/**
+ * The default size for divider is medium.
+ */
 export const Default = DividerGroup.bind({});
 Default.args = {};
 
+// ********* DOCS ONLY ********* //
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When a vertical divider is used inside of a flex container, use `align-self: stretch; block-size: auto;` on the divider.
+ */
+export const VerticalGroup = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+VerticalGroup.storyName = "Vertical";
+VerticalGroup.tags = ["!dev"];
+VerticalGroup.args = {
+	vertical: true,
+};
+VerticalGroup.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticWhiteGroup = (args, context) => Sizes({
```

Comment:
There's no size-based changes that happen when the static white variant is used so it might not make sense for us to push this test through Sizes and perhaps use the same template as Default so that we're capturing any future states.

Reply:
Oh ok, sure. The [docs site](https://opensource.adobe.com/spectrum-css/divider.html) displayed all three sizes, so I was trying to replicate that. But making sure these static colors represent everything on the testing grid is more valuable, for sure.

Reply:
Here's the commit with this update: https://github.com/adobe/spectrum-css/pull/2833/commits/b84e78e7e5533ac6b9f4d67b31583bc9c5718cc1